docs/MigrationTimeline.md

```diff
@@ -0,0 +1,101 @@
+# Migration Timelines
+
```

Comment:
I agree w/ @peternied's comments; we could use a preamble here to ease the reader into the content which they are about to consume given its technical nature.  Some context would be great (e.g. purpose of this section, assumptions it makes, how the reader should use it for their own planning, etc).

Reply:
Updated, thanks